src/app/eslint/index.js

```diff
@@ -6,7 +6,8 @@ import {
   installDependencies
 } from 'src/utils/npm'
 import { actionText, dim } from 'src/utils/log-style'
-import { askLintChoices, getLintChoices } from 'src/utils/prompts'
+import { askLintChoices, getLintChoices, askHookChoices } from 'src/utils/prompts'
+import { githook } from '../githook/index'
```

Comment:
Could we try to avoid using the relative paths unless the files are directly under the directory?

```suggestion
import { githook } from 'src/app/githook/index'
```